frameworks/Java/quarkus/run_quarkus.sh

```diff
@@ -16,6 +16,7 @@ JAVA_OPTIONS="-server \
   -Dio.netty.buffer.checkBounds=false \
   -Dio.netty.buffer.checkAccessible=false \
   -Djava.util.logging.manager=org.jboss.logmanager.LogManager \
+  -Dquarkus.http.idle-timeout=0 \
```

Comment:
I don't remember - wasn't this made the default?

Reply:
Looking at the quarkus doc, it doesn't seem the case (and flamegraphs agree on that too)


https://github.com/quarkusio/quarkus/blob/f92772992f88718c71e5904aea3393c30beff203/extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpConfiguration.java#L125

Said 30M. Probably the http read idle timeout is zero, instead, I didn't checked

Reply:
oh right, sorry mixed it up with a different property.